src/commands/command_factory.py

```diff
@@ -0,0 +1,27 @@
+from create_command import *
+
+# from set_command import *
+# from get_command import *
+# from delete_command import *
+
+
+class CommandFactory:
+    def __init__(self, input_adaptor):
+        self.input_adaptor = input_adaptor
+        self.available_commands = {"create", "set", "get", "delete"}
+        self.validate()
+        
+    def create(self):
+        cmd = str(self.input_adaptor.command).lower()
```

Comment:
Where ‘ else’ ? It should return wrong command

Reply:
Here if the input is wrong the validate function will know it and raise the error so no need to do it here